ocaml/testsuite/tests/pattern-guards/jane_test.ml

```diff
@@ -0,0 +1,36 @@
+(* TEST
+   * expect *)
+
```

Comment:
This could do with a comment explaining why these tests live in a separate file.

Reply:
Definitely, added one!